src/components/EligibilityForm/components/EligibilityForm.styled.tsx

```diff
@@ -110,9 +124,34 @@ export const ContactFormResultMessage = styled.div<{
   }
 `;
 
-export const ContactFormEligibilityMessage = styled.div`
+export const ContactFormEligibilityMessage = styled.div<{
+  cardMode?: boolean;
+}>`
   font-size: 20px;
   line-height: 1.5;
-  margin-top: 2em;
   color: #000074;
+  ${({ cardMode }) =>
+    !cardMode &&
+    css`
+      margin-top: 2em;
```

Comment:
We have a lot of mix between `em` `rem` and `px`
do you think it's possible to replace your change with a px value ?

Reply:
Sure, if you prefer.

Reply:
I do prefere, but it's still mixed

Reply:
The component's 'styled-file' are now convert from 'em' to 'px'.